### ISSUE TITLE
Benchmark run fails due to error: can not perform a '--user' install

### ISSUE THREAD

**Issue by el8** [mlcommons/cm4mlops#100]
Trying to run bert I'm hitting this error in nvidia's scripts:

Copying mlperf_loadgen.egg-info to build/bdist.linux-x86_64/wheel/mlperf_loadgen-4.0.egg-info
running install_scripts
creating build/bdist.linux-x86_64/wheel/mlperf_loadgen-4.0.dist-info/WHEEL
creating 'dist/mlperf_loadgen-4.0-cp310-cp310-linux_x86_64.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
adding 'mlperf_loadgen.cpython-310-x86_64-linux-gnu.so'
adding 'mlperf_loadgen-4.0.dist-info/METADATA'
adding 'mlperf_loadgen-4.0.dist-info/WHEEL'
adding 'mlperf_loadgen-4.0.dist-info/top_level.txt'
adding 'mlperf_loadgen-4.0.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
ERROR: Can not perform a '--user' install. User site-packages are not visible in this virtualenv.
make[1]: *** [Makefile.build:346: build_loadgen] Error 1
make[1]: Leaving directory '/home/jglauber/CM/repos/local/cache/4f4c6af97c2045cd/repo/closed/NVIDIA'
make: *** [/home/jglauber/CM/repos/local/cache/4f4c6af97c2045cd/repo/closed/NVIDIA/Makefile.build:184: build] Error 2

CM error: Portable CM script failed (name = build-mlperf-inference-server-nvidia, return code = 256)

Running as normal user or root user didn't solve it.

cm-repro logs:
https://gist.github.com/el8/2654d3638885bd3ac3d68782beecacea
https://gist.github.com/el8/ec3cafc4acd8ff3512943c33c00f6b28

**Comment by el8** [comment]
Thanks, I'll retry with that instructions.

One question I had though - is the docker container mandatory to run the nvidia-closed benchmarks?

**Comment by arjunsuresh** [comment]
Yes, some of the Nvidia dependencies require the docker. It is technically possible to make them work outside of docker but that'll require manual download of cudnn, tensorrt etc.

**Comment by el8** [comment]
Sorry for the delay, had to wait for to get the hardware back. I can confirm that using the instructions you pointed out and the nvidia container the --user error is gone.

**Comment by arjunsuresh** [comment]
That's great to know. I'm closing this ticket.
